index.html

```diff
@@ -850,18 +841,26 @@ <h3>
         <ol class="algorithm" data-link-for="WakeLock">
           <li>Let |type| be |lock|'s <a>type</a>.
           </li>
+          <li>Let |record| be the <a>platform wake lock</a>'s <a>state
```

Comment:
Please give record a type.

Reply:
state record is more like an abstract things right now... how do you prefer that I fix this?